backend/level-3/exercise.spec.ts

```diff
@@ -34,7 +33,9 @@ const app = express();
  * Each contract has an address (string), a name (string) and an ABI (list of objects).
  */
 interface IContract {
-    // TODO: define here
+    address: String,
+    name: String,
+    ABI : Object[]
```

Comment:
Object accepts everything and it's pretty bad. You may wanna use Record<string, ??something??> instead